src/Components/ActivationKeysTable/ActivationKeysTable.js

```diff
@@ -14,9 +14,62 @@ const ActivationKeysTable = (props) => {
     role: 'Role',
     serviceLevel: 'SLA',
     usage: 'Usage',
+    updatedAt: 'Updated Date',
   };
   const { isLoading, error, data } = useActivationKeys();
+  console.log(data);
+  const [activeSortIndex, setActiveSortIndex] = React.useState(null);
+  const [sortedData, setSortedData] = React.useState([]);
+  const [activeSortDirection, setActiveSortDirection] = React.useState(null);
   const location = useLocation();
+  React.useEffect(() => {
+    if (data) {
+      setSortedData(data);
+    }
+  }, [data]);
+
+  const getSortParams = (columnIndex) => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection,
+      defaultDirection: 'asc',
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+      const sorted = [...sortedData].sort((a, b) => {
+        const aValue = a[columnNames[index]];
+        const bValue = b[columnNames[index]];
+        if (index === 4) {
+          const aDate = new Date(a.updatedAt);
+          const bDate = new Date(b.updatedAt);
+          if (direction === 'asc') {
+            return aDate - bDate;
+          } else {
+            return bDate - aDate;
+          }
+        }
+        if (aValue < bValue) return direction === 'asc' ? -1 : 1;
+        if (aValue > bValue) return direction === 'asc' ? 1 : -1;
+        return 0;
+      });
```

Comment:
Polish: I think there's merit to putting this function in a dateHelpers.js utility file. It's conceivable that we'll need to sort by dates in other places.